nvflare/fuel/utils/config.py

```diff
@@ -50,6 +57,9 @@ def get_format(self) -> ConfigFormat:
         """
         return self.format
 
+    def get_exts(self) -> List[str]:
+        return self.format.extensions(self.format)
```

Comment:
this is class format, suggest calling it using `ConfigFormat.extensions(self.format)` for clarity